airflow/www/templates/airflow/dags.html

```diff
@@ -350,7 +360,7 @@ <h2>DAGs</h2>
               })
               .on('click', function(d, i) {
                   if (d.count > 0)
-                    window.location = "{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id=" + d.dag_id + "&_flt_3_state=" + d.state;
+                    window.location = BASE_URL_DAG_RUN_LIST + "$?_flt_3_dag_id=" + d.dag_id + "&_flt_3_state=" + d.state;
```

Comment:
Can dag ids contain characters that need to be url-encoded? Might be safer to build query strings with `$.param` or similar.

Reply:
I will add your comment to the list of my tasks. I will try to solve this problem globally for the entire Airflow instead of creating a change only locally. Local changes are not permanent and only a global change will avoid problems in the future. This is only the first PR in this area. I want to make a number of changes to improve the entire Javascript code and user interface. I agree that this change is necessary. However, I have not yet chosen how to solve this problem. I will analyze your suggestions to use jquery, but I do not know if this will be the best solution. The Javascript community drops jquery because it is outdated, big, hard, slow. It is a monolithic library that was created during the old days of Javascript   There were no module bundles. The use of large monolitic libraries was the only way to effectively provide libraries. JQuery was also the only way to solve the browser incompatibility problem. Now there are better solutions to this problem - polyfill, babel, etc. 

See: https://blog.getbootstrap.com/2019/02/11/bootstrap-4-3-0/#branches-hugo-and-jquery